catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -109,6 +113,17 @@ public void init() {
           res.header("Content-Encoding", "gzip");
         });
 
+    get(
+        "/cql/transforms",
+        (req, res) -> {
+          List transformers =
```

Comment:
❓ Raw List?

Reply:
Since this is rendered as JSON, typing is irrelevant here.